examples/README.md

```diff
@@ -263,6 +264,79 @@ to 256
 
 [hpc-enterprise-slurm.yaml]: ./hpc-enterprise-slurm.yaml
 
+## [fsi-montecarlo-on-batch.yaml](../community/examples/fsi-montecarlo-on-batch.yaml) ![community-badge]
+## Monte Carlo Simulations for Value at Risk
+This blueprint will take you through a tutorial on an FSI Value at Risk calculation using Cloud tools:
+* Batch
+* Pub/Sub
+  * BigQuery pubsub subscription
+* BigQuery
+* Vertex AI Notebooks
+
+See the [full tutorial here](../docs/tutorials/fsi-montecarlo-on-batch/README.md).
```

Comment:
I would recommend a single source of truth. Just point to the instructions and don't repeat deployment instructions below.

Reply:
Done